### PR TITLE
Turganov HW2

### PR DESCRIPTION
**Pull request by 1MRUN**

### CODE REVIEW

File: Main.hs [hunk 1]
```diff
@@ -11,28 +11,31 @@ import qualified Data.List as L
...
+    | otherwise = foldr (*) 1 (zipWith (-) (a) (tail a))
 
 isSorted :: [Int] -> Bool 
-isSorted xs = undefined
+isSorted a = and (zipWith (<=) a (tail a))
 
 countElement :: Int -> [Int] -> Int 
-countElement = undefined
+countElement a x = length(filter (check a) x)
+    where check eq var = (eq == var)
```
Comment (alex28sh) [inline Main.hs:28]: actually you can use prefix notation of equality operator as a parameter for filter, without introducing check function

a == b ~ ~ `==` a b

Comment (alex28sh) [reply Main.hs:28]: try to redo this task

**Review by alex28sh — CHANGES_REQUESTED** [review]
7/8

**Comment by kajigor** [comment]
Please make sure to put your name in the PR title next time.

File: Main.hs (hunk 1 above)
Comment (1MRUN) [reply Main.hs]: > try to redo this task

Changed the code.

**Comment by kajigor** [comment]
Ok.

**Pull request closed without merging**